public/locales/translation.xx.json

```diff
@@ -0,0 +1,5 @@
+{
+  "appHeader": {
+    "giveFeedback": "^Give feedback$"

```

Comment:
Something to consider for later. Perhaps we can wrap that t function with our own to hook into the calls. If it is xx, then take the en string and add ^$. That way we can avoid having this file.